tests/test_models.py

```diff
@@ -85,3 +85,25 @@ def test_farm(self):
         sau.save()
 
         self.assertEqual(f.name, sau.farm.name)
+
+    def test_batch(self):
+        britanna, rambo = get_sheep(('britanna', 'rambo'))
+        self.assertEqual(9, len(Sheep.objects.all()))
```

Comment:
This creates an unfortunate dependency on the test data. We should assert this once, somewhere else.